### PR TITLE
Feat/server basic post route

### PR DESCRIPTION
**Pull request by tmkuba**
https://trello.com/c/ToIRGrTw

trello card says it sends right shape to theoretical DB function.  Right now I just console.log out the request JSON data, which should already be in the right shape to go directly to the DB helper function

### CODE REVIEW

**Comment by leungnamson1023** [comment]
@tmkuba Hey Mark, it says that it has a conflict, but I am going to review it anyways

**Comment by leungnamson1023** [comment]
It's just for package.json

File: server/routers/reservations.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const router = express.Router();
+
+// middleware that is specific to this router
+router.use(bodyParser.json());
```
Comment (leungnamson1023) [inline server/routers/reservations.js:7]: that's a cool trick

**Review by leungnamson1023 — COMMENTED** [review]
looks good!

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "./server/server.js",
   "scripts": {
     "test": "./node_modules/.bin/jest",
-    "test:watch": "./node_modules/.bin/jest -w",
+    "test:watch": "./node_modules/.bin/jest --watch",
```
Comment (matgbx) [inline package.json:8]: does this make jest actively watch your test files for changes?

File: server/routers/reservations.js (hunk 1 above)
Comment (matgbx) [reply server/routers/reservations.js:7]: Yea, that is cool

**Review by matgbx — APPROVED** [review]
Everything looks good here

File: package.json (hunk 2 above)
Comment (tmkuba) [reply package.json:8]: Yeah, I thought the flag was -w but it is actually --watch.  jest will auto-re-run when it notices changes in any of your files.

**Pull request merged**